packages/lit-element/src/lib/css-tag.ts

```diff
@@ -31,24 +29,20 @@ export class CSSResult {
         'CSSResult is not constructable. Use `unsafeCSS` or `css` instead.'
       );
     }
-
     this.cssText = cssText;
   }
 
   // Note, this is a getter so that it's lazy. In practice, this means
   // stylesheets are not created until the first element instance is made.
-  get styleSheet(): CSSStyleSheet | null {
-    if (this._styleSheet === undefined) {
-      // Note, if `supportsAdoptingStyleSheets` is true then we assume
-      // CSSStyleSheet is constructable.
-      if (supportsAdoptingStyleSheets) {
-        this._styleSheet = new CSSStyleSheet();
-        this._styleSheet.replaceSync(this.cssText);
-      } else {
-        this._styleSheet = null;
-      }
+  get styleSheet(): CSSStyleSheet | undefined {
+    let sheet: CSSStyleSheet | undefined;
```

Comment:
Why remove the memoization? The cache to reuse the same `CSSResult` is only really useful if you can reuse the stylesheet as well, right?

Reply:
That was a mistake. Putting it back.